Makefile.features

```diff
@@ -0,0 +1,8 @@
+# import list of provided features
+-include $(RIOTBOARD)/$(BOARD)/Makefile.features
+-include $(RIOTCPU)/$(CPU)/Makefile.features
+
+DEFAULT_FEATURES += periph_pm
+
+# add available default features to required list
+FEATURES_REQUIRED += $(filter-out $(DISABLE_FEATURES), $(filter $(FEATURES_PROVIDED), $(DEFAULT_FEATURES)))
```

Comment:
```FEATURES_REQUIRED``` is being used to set the defines in riot_build.h.
```DEFAULT_FEATURES``` is used to always include a feature, *if available* (e.g., always require periph_pm)
```DISABLE_FEATURES``` can be used to override ```DEFAULT_FEATURES```(e.g., this time, build without periph/pm)

The file adds all features from "default features" that are available, unless they are in "disabled features", to the required list.

So in set operations, ``` FEATURES_REQUIRED += (DEFAULT_FEATURES | FEATURES_PROVIDED) - DISABLED_FEATURES ```. I think the last line in the file does that.